README.adoc

```diff
@@ -99,7 +99,7 @@ During the course of content development, writers merge branches with changes ei
 
 Github actions create Git tags and releases that appear at the repo’s link:https://github.com/starknet-community-libs/starknet-docs/releases[Releases] and link:https://github.com/starknet-community-libs/starknet-docs/tags[Tags] pages.
 
-When a feature branch is merged into the `master` branch, a github action creates a tag and release, and updates `CHANGELOG.md`. It then publishes the new content to docs.starknet.io.
+When a feature branch is merged into the `master` branch, a github action creates a tag and releases, and updates `CHANGELOG.md`. It then publishes the new content to docs.starknet.io.
```

Comment:
Actually, it was talking about a Github release, not the verb _release_.

The truth is that this part of the flow is not useful for docs and I intend to remove it from our workflow.

```suggestion
When a feature branch is merged into the `master` branch, a github action creates a release tag in the format `v<version>.<major_update>.<minor_update>` and updates `CHANGELOG.md`. It then publishes the new content to docs.starknet.io.
```